ts/components/ui/cards/payment/PaymentCardBig.tsx

```diff
@@ -0,0 +1,220 @@
+import React from "react";
+import { StyleSheet, View } from "react-native";
+import Placeholder from "rn-placeholder";
+import { WithTestID } from "../../../../types/WithTestID";
+import { formatDateAsLocal } from "../../../../utils/dates";
+import { IOLogoPaymentType, LogoPayment } from "../../../core/logos";
+import { VSpacer } from "../../../core/spacer/Spacer";
+import { LabelSmall } from "../../../core/typography/LabelSmall";
+import { NewH6 } from "../../../core/typography/NewH6";
+import { IOColors } from "../../../core/variables/IOColors";
+import { IOBannerRadius } from "../../../core/variables/IOShapes";
+import { IOStyles } from "../../../core/variables/IOStyles";
+import { LogoPaymentOrDefaultIcon } from "../../utils/baseComponents/LogoPaymentOrDefaultIcon";
+import I18n from "../../../../i18n";
+
+const styles = StyleSheet.create({
+  cardContainer: {
+    flex: 1,
+    alignItems: "flex-start",
+    alignContent: "space-between",
+    justifyContent: "space-between",
+    height: 207,
+    borderRadius: IOBannerRadius,
+    backgroundColor: IOColors["grey-100"],
+    padding: 24
+  },
+  bottomRow: {
+    height: 48,
+    width: "100%",
+    flexDirection: "row",
+    alignItems: "center",
+    justifyContent: "space-between"
+  }
+});
+
+type PAYMENT_METHOD_CARD_TYPES = {
+  CREDIT: "CREDIT";
+  PAGOBANCOMAT: "PAGOBANCOMAT";
+  PAYPAL: "PAYPAL";
+  COBADGE: "COBADGE";
+  BANCOMATPAY: "BANCOMATPAY";
+};
```

Comment:
Is this type really needed? 

At line 56, in `PaymentCardStandardProps`, you are using this object to define a discriminated type but the type itself should be extracted from `PaymentCardStandardProps`, and not the opposite.

Also, Pascal Case is preferred for type names.

I suggest to change the type as follows:

```suggestion
type PaymentMethodCardsType = PaymentCardStandardProps["type"];
```

Reply:
that type stood there as a remain of a previous implementation I was working on, as of now we can just remove it and it should be all the same, with string costants 👍

Reply:
addressed in [aeb2527](https://github.com/pagopa/io-app/pull/4759/commits/aeb2527c92241119b9053c2ba1572f01acef402b)